lib/_range.py

```diff
@@ -62,6 +62,19 @@ def __repr__(self):
             return "%s(%r, %r, %r)" % (self.__class__.__name__,
                 self._lower, self._upper, self._bounds)
 
+    def __str__(self):
+        if (self._lower, self._upper, self._bounds) == (None, None, None):
+            return 'empty'
```

Comment:
I think this can be:

    if self._bound is None:
        return 'empty'

Reply:
No worries, I'll fix this myself and merge :)

Reply:
Thanks :smile: 

It was my best guess after digging through the code.